### DISCUSSION TITLE
Taiko mod field reversal

### DISCUSSION THREAD

**Discussion by cdwcgt** [ppy/osu#22496] [Ideas]
![image](https://user-images.githubusercontent.com/34775378/216324805-3b66d70c-44b1-4542-b84d-2537886761d2.png)

**Comment by bdach** [comment]
Why would this be a taiko mod if mania scrolling direction is a ruleset setting?

**Reply by cdwcgt** [reply]
Why FF is a mod or not a settings in ctb

**Reply by bdach** [reply]
Good question.

I have no idea which one of those is the right approach for taiko.

**Reply by cdwcgt** [reply]
I think for mania, stable itself has a setting there, so lazer has the same behavior.
but for taiko and ctb, there are new things in lazer. So put them to **Fun** mod

**Reply by mk56-spn** [reply]
> I think for mania, stable itself has a setting there, so lazer has the same behavior.
> but for taiko and ctb, there are new things in lazer. So put them to **Fun** mod

This is a bad idea imo. Your just creating mods for the sake of it at that point which don't need to exist and make the game messier

**Reply by cdwcgt** [reply]
If so, why does the ff of ctb exist?

**Reply by mk56-spn** [reply]
> If so, why does the ff of ctb exist?

I also disagree with that existing. Std is different since HR already had interactions with flipping screens

**Comment by TheFearlessDeath500** [comment]
I’ve seen people mod the official Taiko games to do this, this could probably work— I just don’t see a lot of people using it.
